README.md

```diff
@@ -1 +1,179 @@
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Landing Page <a name="about-project"></a>
+
+**Landing Page** for trials.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">N/A</a></li>
+
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Landing page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy of this project and run it in your computer, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- A working computer.
+- Connection to internet.
+
+### Setup
+
+To get a local copy up and running follow these simple example steps.
+
+Clone this repository in the desired folder:
+```
+cd my-folder
+git clone https://github.com/jmonto55/CR-trialspast-project.git
+```
+
+### Install
+
+To install this project:
+```
+cd CR-trialspast-project
+code .
+```
+
+### Usage
+
+To run the project, execute the following:
+
+```
+right click, Open Live server on index.html
+```
+
+### Run Tests
+
+To run the project tests, execute the following command:
+
+```
+N/A
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Jose Montoya**
+
+- GitHub: [@jmonto55](https://github.com/jmonto55)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project feel free to frok it and use it as you need.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the information provided to build this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Can I use the project for any purpose?**
+
+  - Yes, you can use this files for anything you need.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Your license link is broken, actually you didn't even add the License file

Reply:
- [ ] I think it's better if you suggest changes instead of giving orders when requesting changes. This way, students will experience less frustration when implementing the changes.

- [ ] Please always explain the reason for the changes so that students will understand why they need to make them. This will also make students look forward to their next reviews with optimism :smile:.